server/svix-server/src/worker.rs

```diff
@@ -47,42 +48,42 @@ const USER_AGENT: &str = concat!("Svix-Webhooks/", env!("CARGO_PKG_VERSION"));
 const OP_WEBHOOKS_SEND_FAILING_EVENT_AFTER: usize = 4;
 
 /// A simple struct noting the context of the wrapped [`DateTimeUtc`]. This struct is returned when
-/// you are to disable disable an endpoint. This is optionally returned by [`process_failure_cache`]
+/// you are to disable disable an endpoint. This is optionally returned by [`process_failure_store`]
 /// which is to be called after all retry events are exhausted.
 #[repr(transparent)]
 struct EndpointDisableInfo {
     first_failure_at: DateTimeUtc,
 }
 
 /// The first_failure_at time is only stored in Postgres after the endpoint has been disabled.
-/// Otherwise, it is stored in the cache with an expiration.
+/// Otherwise, it is stored in the store with an expiration.
 #[derive(Deserialize, Serialize)]
-pub struct FailureCacheValue {
+pub struct FailureValue {
     pub first_failure_at: DateTimeUtc,
 }
 
-kv_def!(FailureCacheKey, FailureCacheValue, "SVIX_FAILURE_CACHE");
+store_kv_def!(FailureKey, FailureValue, "FAILURES_");
```

Comment:
Missing the Svix prefix.

After seeing this, I'm 100% convinced about my other comment of splitting the prefix being confusing.